infra/dev/postgres/initScript/init.sql

```diff
@@ -0,0 +1,12 @@
+CREATE DATABASE laravel;
```

Comment:
This file can be deleted since db will be managed by Laravels ORM.

I'm guessing it was for testing purposes.

The Larvel command to create the db is 
```
$ php artisan migrate
```

Reply:
yes

Reply:
Fixed in 71045c0